core/src/main/java/nu/yona/server/analysis/entities/WeekActivityRepository.java

```diff
@@ -0,0 +1,18 @@
+package nu.yona.server.analysis.entities;
+
+import java.time.ZonedDateTime;
+import java.util.UUID;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface WeekActivityRepository extends CrudRepository<WeekActivity, UUID>
```

Comment:
Remove "zoned" everywhere. ``ZonedDateTime`` needs to become our new (and thus common) date class.